src/jit/hwintrinsicxarch.cpp

```diff
@@ -493,7 +489,7 @@ GenTree* Compiler::impHWIntrinsic(NamedIntrinsic        intrinsic,
     // table-driven importer of simple intrinsics
     if (impIsTableDrivenHWIntrinsic(category, flags))
     {
-        if (!varTypeIsSIMD(retType) || (flags & HW_Flag_BaseTypeFromArg))
+        if (!varTypeIsSIMD(retType) || (flags & HW_Flag_BaseTypeFromArg) != 0)
```

Comment:
Please parenthesize sub-expressions. This is specified in the coding conventions: https://github.com/dotnet/coreclr/blob/master/Documentation/coding-guidelines/clr-jit-coding-conventions.md#111-logical-and-arithmetic-expressions but not enforced by the formatter (and often missed by developers and reviewers, including me):
E.g.:
```
if (!varTypeIsSIMD(retType) || ((flags & HW_Flag_BaseTypeFromArg) != 0))
```

Reply:
Got it, will do.